Dockerfile

```diff
@@ -8,11 +8,26 @@ ENV LANGUAGE en_US.UTF-8
 
 RUN apt-get -q update &&\
   apt-get -qy --force-yes dist-upgrade &&\
-  apt-get install -qy --force-yes forked-daapd avahi-daemon &&\
+  apt-get install -qy --force-yes apt-utils forked-daapd avahi-daemon
+
+RUN apt-get install -qy --force-yes wget build-essential git autotools-dev autoconf libtool gettext gawk gperf antlr3 libantlr3c-dev libconfuse-dev libunistring-dev libsqlite3-dev libavcodec-dev libavformat-dev libavfilter-dev libswscale-dev libavutil-dev libasound2-dev libmxml-dev libgcrypt11-dev libavahi-client-dev zlib1g-dev libevent-dev libcurl4-openssl-dev
+  
+RUN wget -q -O - https://apt.mopidy.com/mopidy.gpg | sudo apt-key add - &&\
+  wget -q -O /etc/apt/sources.list.d/mopidy.list https://apt.mopidy.com/jessie.list &&\
+  apt-get update &&\
+  apt-get install -qy --force-yes libspotify-dev python-spotify &&\
   apt-get clean &&\
   rm -rf /var/lib/apt/lists/* &&\
   rm -rf /tmp/*
 
+RUN git clone https://github.com/ejurgensen/forked-daapd.git &&\
```

Comment:
This will get the bleeding edge of forked-daaped. Is there a released version that has the spotify, lastfm and mpd capabilities that can be referenced here? At the least the container can be tagged with the version so there is something stable available.